compiler/testData/diagnostics/tests/declarationChecks/MultiDeclarationErrors.fir.kt

```diff
@@ -11,7 +11,7 @@ class MyClass2 {}
 <!CONFLICTING_OVERLOADS!>fun MyClass2.component1()<!> = 1.3
 
 fun test(mc1: MyClass, mc2: MyClass2) {
-    val (<!INAPPLICABLE_CANDIDATE!>a<!>, b) = <!COMPONENT_FUNCTION_MISSING!>mc1<!>
+    val (a, b) = <!COMPONENT_FUNCTION_MISSING!>mc1<!>
```

Comment:
Perhaps, this is the only regression. In old FE, `a` has `OPERATOR_MODIFIER_REQUIRED` as well as `COMPONENT_FUNCTION_MISSING`. FIR should report unresolved error rather than inapplicable candidate. :\

https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/declarationChecks/MultiDeclarationErrors.kt#L14